docs/getting-started/nullblk.md

```diff
@@ -150,15 +158,20 @@ nulldev=$(create_zoned_nullb $1 $2 $3 $4)
 echo "Created /dev/nullb$nulldev"
 ```
 
-This script (*nullblk-zoned.sh*) takes four arguments: the sector size in bytes,
-of the emulated device, the device zone size in MiB, the number of conventional
-zones (which can be 0) and the number of sequential write required zones. Memory
-backing for writen sectors is turned on with this script (memory_backed=1).
-This enables run-time persistancy of the data written to the sectors of the
-emulated device. The writen data is lost when the emulated device is destroyed.
+This script (*nullblk-zoned.sh*) takes four arguments: 
+
+   1. the sector size in bytes of the emulated device
+   2. the device zone size in MiB 
+   3. the number of conventional zones (which can be 0) 
+   4. the number of sequential write required zones. 
+
+Memory-backing for written sectors can be turned on with this script
+(memory_backed=1). This enables run-time persistence of the data written to
+the sectors of the emulated device. The writen data is lost when the emulated
+device is destroyed.
 
 For example, a small zoned device with 4 conventional zones and 8 sequential
-write required zones of 64 MiB can be created with the following command.
+write-required zones of 64 MiB can be created with the following command:
```

Comment:
Please drop the hyphen here. I understand that it makes the sentence more grammatically correct, but "sequential write required" is used everywhere and that is the wording in the standards document too. So I prefer sticking to it, even though it is not perfect.

Reply:
Would you object to the addition of quotation marks around the phrase "sequential write required", as in this sentence?

Reply:
No strong objection, but:
1) This will look a little strange since "conventional zones" will not get the quotation (no need, right ? That is correct simple English adj+noun... no ?)
2) The standards do not use quotation, and since I would prefer staying as close as possible to the standards wording/style, adding quotation would not be ideal.

I think that we keep coming back to these hackish-english problems because we do not really have a good place where these idioms (? is that what we should call that ?) are clearly defined, that is, zone types are not defined clearly anywhere ! Looking at the zoned storage overview (https://zonedstorage.io/introduction/zoned-storage/), we never define zone types. The SMR and ZNS overview do not have clear definitions of zone types either. That should be fixed. I will draft something next week.

Reply:
I've been thinking about this for the past 72 hours, and I am going, provisionally, to adopt this convention of not insisting on complex adjectives with hyphens. At least, not yet.

I can sense that I remain fairly ignorant of the terms relevant to zoned block devices, and that sense makes me think that I should wait a few months before insisting on complex adjectives with hyphens. After I've seen these terms in their natural environment for longer, I'll have a keener sense of when complex adjectives are appropriate and when, as probably here, they are not.

For posterity (for me, in a month or two): my early articulate ignorance cries out for the guidance of complex adjectives when I read this, but I have enough experience of the process of familiarizing myself with a technology to know that what today looks like meaningless string of nouns tomorrow looks like a fixed phrase with an obvious reference. For example: who could know except by experience that "closed captioning" means "subtitles for the deaf"? ("Closed captioning" is here contrasted with "open captioning", which means "subtitles that cannot be switched off or made to be invisible, because 'open captioning' subtitles are meant to be a part of the filmic experience".) (However, the fact that I have put open captioning in single-quotes in the sentence before this one argues that some kind of complex-adjective- or complex-noun- marker might be helpful in situations like this, and the fact that I have omitted any single-quote marker in this sentence establishes the converse of that argument.)

Reply:
I have changed the text so that it reads the way you prefer, Damien.